src/main/java/duck/task/TodoTask.java

```diff
@@ -0,0 +1,48 @@
+package duck.task;
+
+import duck.DuckException;
+
+/**
+ * Represents a todo task.
+ */
+public class TodoTask extends Task {
+    /**
+     * Creates a new TodoTask.
+     * 
+     * @param name   Name of the task.
+     * @param isDone Status of the task.
+     */
+    public TodoTask(String name, boolean isDone) {
+        super(name, isDone);
+    }
+
+    @Override
+    public String stringify() {
+        return "T" + super.stringify();
```

Comment:
Perhaps you could change the naming of this function because having a stringify and a toString function together may seem a little confusing.